app/workers/gamedev_worker.rb

```diff
@@ -1,19 +1,18 @@
 class GamedevWorker
-	include Sidekiq::Worker
+  include Sidekiq::Worker
 
-	def perform
-		response = RubyStackoverflow.questions({ order: 'asc', filter: 'withbody', pagesize: 100 })
-		response.data.each do | question|
-		  p =  ExternalPost.new(title: question.title,
-                            user_display_name: question.owner[:display_name],
-                            body_html: question.body,
-                            source_url: question.link,
-                            external_id: question.question_id)
-		  p.save
-	    end
-	end
+  def perform
+    response = RubyStackoverflow.questions({ order: 'asc', filter: 'withbody', pagesize: 100 })
```

Comment:
Line is too long. [95/80]<br>Redundant curly braces around a hash parameter.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.